virtcontainers/device/config/config.go

```diff
@@ -213,11 +233,20 @@ var GetHostPathFunc = GetHostPath
 // GetHostPath is used to fetch the host path for the device.
 // The path passed in the spec refers to the path that should appear inside the container.
 // We need to find the actual device path on the host based on the major-minor numbers of the device.
-func GetHostPath(devInfo DeviceInfo) (string, error) {
+func GetHostPath(devInfo DeviceInfo, vhostUserStoreEnabled bool) (string, error) {
 	if devInfo.ContainerPath == "" {
 		return "", fmt.Errorf("Empty path provided for device")
 	}
 
+	sysDevPrefix := SysDevPrefix
+	devDirPrefix := "/dev"
+
+	// Filter out vhost-user storage devices by device Major numbers.
+	if vhostUserStoreEnabled && (devInfo.Major == VhostUserSCSIMajor || devInfo.Major == VhostUserBlkMajor) {
+		sysDevPrefix = VhostUserMknodDir
```

Comment:
I don't follow, who is going to populate these directories? can you document it?

Reply:
Sure, let me add the details to a document